diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -548,7 +551,7 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
 
     @Test
     @Tag(WarningNames.WRONG_NEWLINES)
-    fun `should trigger on non-multiline lambdas`() {
+    fun `shouldn't trigger on non-multiline lambdas`() {
```

Comment:
Why did you remove it?

Reply:
Because in this case there are no mistakes.